materials/r-sql.md

```diff
@@ -5,11 +5,17 @@ title: Integrating R and SQL
 language: R
 ---
 
-* Already seen how to connect to database using `dplyr`
-* What if we want to do this outside of a `dplyr` context
-* Use `DBI` and `RSQLite` (or other appropriate DBMS plugin)
+> Remember to
+>    
+> * download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171). *Make sure it does not have `SpeciesCounts` view.*
```

Comment:
I added this note after an experience I had testing the lecture. When I ran the code a second time, I got an error that the `SpeciesCounts` already existed. Could other instructors run into this same issue? Do they need a reminder to get a fresh version?